src/lib/services/ollama.ts

```
@@ -19,3 +19,74 @@ export async function getOllamaModels(): Promise<Result<OllamaModel[]>> {
+
+			const chunk = decoder.decode(value, { stream: true });
+			const lines = chunk.split('\n').filter((line) => line.trim() !== '');
+
+			for (const line of lines) {
+				try {
+					const parsed = JSON.parse(line);
+
+					// Debug log to see structure
+					console.log('Raw parsed chunk:', parsed);
+
+					if (parsed.message && parsed.message.content) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)